lib/js/jsre.nim

```diff
@@ -0,0 +1,43 @@
+## Regular Expressions for the JavaScript target.
+## * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
+
+type RegExp* {.importjs.} = object    ## Regular Expressions for JavaScript target.
+  flags* {.importjs.}: cstring        ## cstring that contains the flags of the RegExp object.
+  dotAll* {.importjs.}: bool          ## Whether `.` matches newlines or not.
+  global* {.importjs.}: bool          ## Whether to test the regular expression against all possible matches in a string, or only against the first.
+  ignoreCase* {.importjs.}: bool      ## Whether to ignore case while attempting a match in a string.
+  multiline* {.importjs.}: bool       ## Whether to search in strings across multiple lines.
+  source* {.importjs.}: cstring       ## The text of the pattern.
+  sticky* {.importjs.}: bool          ## Whether the search is sticky.
+  unicode* {.importjs.}: bool         ## Whether Unicode features are enabled.
+  lastIndex* {.importjs.}: cint       ## The lastIndex is a read/write integer property of regular expression instances that specifies the index at which to start the next match.
```

Comment:
I promise this is my last nit! (sorry... thanks for your patience)

keep docs DRY:
* no need to repeat the field name in its docstring (unless it'd lead to ambiguities)
* no need to mention `integer property` since the type is apparent in the docs (`: cint`)
* no need to specify `of regular expression instances` since we're already inside `type RegExp*`

so I'd simply write the shorter/more readable:
```
## index at which to start the next match (read/write property)
```
likewise, the doc comment for `input` can be shortened a bit.